aidies-project/client/src/components/CampaignCreate.js

```diff
@@ -44,32 +44,28 @@ const CampaignCreate = ({ needLogin }) => {
     const [closingDate, setClosingDate] = useState('2020-10-26T18:36');
     const [charity, setCharity] = useState(2);
     const [category, setCategory] = useState(3);
-
+    const [button, setButton] = useState('Start a campaign');
+    
     const classes = useStyles();
     const history = useHistory();
-    const location = useLocation();
+    const dispatch = useDispatch();
+
     const userId = useSelector(state => state.authentication.user.id);
     const charities = useSelector(state => state.campaign.charities);
     const categories = useSelector(state => state.campaign.categories);
-    
-    const dispatch = useDispatch();
+
     useEffect(() => {
         dispatch(getCategories());
         dispatch(getCharities());
     }, []); 
     
-    
-    if (needLogin) {
-        dispatch(setLocation(location.pathname));
-        return <Redirect to="/login" />
-    }
     const handleCreate = (e) => {
         e.preventDefault();
         dispatch(createCampaign({ campaignName, summary, story, startingPrice, closingDate, userId, charity, category }));
         history.push("/"); 
     }
     return (
-        <Card className={classes.root} className="create-card">
+        <Card className={`${classes.root} create-card`} >
```

Comment:
I'm not super familiar with adding additional styling to Material-UI components, but I don't think it's the best practice to mix both a `.css` class and a class from `useStyles`. Try to move the stylings for the `.create-card` css class into `useStyles`!